diff --git a/lib/services/download_manager.dart b/lib/services/download_manager.dart
index bb45e6e..461fdbd 100644
--- a/lib/services/download_manager.dart
+++ b/lib/services/download_manager.dart
@@ -373,11 +373,7 @@ class DownloadManager {
           .getManifest(
         videoId,
         requireWatchPage: true,
-        ytClients: [
-          YoutubeApiClient.ios,
-          YoutubeApiClient.androidVr,
-          YoutubeApiClient.tv,
-        ],
+        ytClients: [YoutubeApiClient.androidVr],
       );
       List<AudioOnlyStreamInfo> streamInfos = manifest.audioOnly
           .where((a) => a.container == StreamContainer.mp4)
diff --git a/lib/services/yt_audio_stream.dart b/lib/services/yt_audio_stream.dart
index 825023d..537ef4e 100644
--- a/lib/services/yt_audio_stream.dart
+++ b/lib/services/yt_audio_stream.dart
@@ -26,11 +26,7 @@ class YouTubeAudioSource extends StreamAudioSource {
     final manifest = await ytExplode.videos.streams.getManifest(
       videoId,
       requireWatchPage: true,
-      ytClients: [
-        YoutubeApiClient.ios,
-        YoutubeApiClient.androidVr,
-        YoutubeApiClient.tv,
-      ],
+      ytClients: [YoutubeApiClient.androidVr],
     );
     Iterable<AudioOnlyStreamInfo> supportedStreams = manifest.audioOnly;
     if (Platform.isMacOS || Platform.isIOS) {
